### PR TITLE
Chore/dev 947 upgrade loki

### PR DESCRIPTION
**Pull request by MacLikorne**

### CODE REVIEW

**Review by benjaminch — APPROVED** [review]

File: lib/digitalocean/bootstrap/chart_values/loki.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    retention_period: 12w
+
+extraPorts:
+  - name: gossip
+    port: 7946
+    protocol: TCP
+    targetPort: 7946
+
+persistence:
+  enabled: false
```
Comment (deimosfr) [inline lib/digitalocean/bootstrap/chart_values/loki.yaml:35]: is it really false?

File: lib/digitalocean/bootstrap/chart_values/loki.j2.yaml [hunk 2]
```diff
@@ -1,255 +0,0 @@
...
-#     topologyKey: "kubernetes.io/hostname"
-
-## StatefulSet annotations
-annotations: {}
-
-# enable tracing for debug, need install jaeger and specify right jaeger_agent_host
-tracing:
-  jaegerAgentHost:
-
-config:
```
Comment (deimosfr) [inline lib/digitalocean/bootstrap/chart_values/loki.j2.yaml:46]: this was the custom one, to check

File: lib/aws/bootstrap/chart_values/loki.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+config:
```
Comment (deimosfr) [inline lib/aws/bootstrap/chart_values/loki.yaml:1]: same

File: lib/aws/bootstrap/chart_values/loki.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  server:
```
Comment (benjaminch) [inline lib/aws/bootstrap/chart_values/loki.yaml:2]:
```
auth_enabled: false
  distributor:
    ring:
      kvstore:
        store: memberlist
  ingester:
    chunk_idle_period: 3m
    chunk_block_size: 262144
    chunk_retain_period: 1m
    max_transfer_retries: 0
    lifecycler:
      ring:
        kvstore:
          store: memberlist
        replication_factor: 1
  limits_config:
    ingestion_rate_mb: 20
    ingestion_burst_size_mb: 30
    enforce_metric_name: false
    reject_old_samples: true
    reject_old_samples_max_age: 168h
  schema_config:
    configs:
      - from: 2020-05-15
        store: boltdb-shipper
        object_store: s3
        schema: v11
        index:
          prefix: index_
          period: 24h
  server:
    http_listen_port: 3100
  storage_config:
    aws:
      s3: ""
      region: ""
      access_key_id: ""
      secret_access_key: ""
    boltdb_shipper:
      active_index_directory: /data/loki/index
      shared_store: s3
      resync_interval: 5s
      cache_location: /data/loki/boltdb-cache
  chunk_store_config:
    max_look_back_period: 12w
  table_manager:
    retention_deletes_enabled: true
    retention_period: 12w
  memberlist:
    abort_if_cluster_join_fails: false
    bind_port: 7946
    join_members:
      - loki-headless.logging.svc:7946
    max_join_backoff: 1m
    max_join_retries: 10
    min_join_backoff: 1s
```

Comment (benjaminch) [reply lib/aws/bootstrap/chart_values/loki.yaml:2]: To be double check what should stay

File: lib/aws/bootstrap/chart_values/loki.yaml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    max_join_retries: 10                  # qovery setting
+    min_join_backoff: 1s                  # qovery setting
+
+extraPorts:
+  - name: gossip
+    port: 7946
+    protocol: TCP
+    targetPort: 7946
+
+persistence:
```
Comment (deimosfr) [inline lib/aws/bootstrap/chart_values/loki.yaml:65]: plz remove as not used

**Review by deimosfr — APPROVED** [review]

**Pull request merged**